From dd6781722e998cf02046241c758101a507a2051f Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 12 Dec 2024 14:04:20 +0800
Subject: [PATCH 283/296] AOSCOS: MIPS: loongson3-memcpy: replace
 <asm/export.h> with <linux/export.h>

Per commit 9259e15b3f27 ("mips: replace #include <asm/export.h> with
for <linux/export.h>.

Follow the commit and revise loongson3-memcpy.S accordingly.

Fixes: "AOSCOS: memcpy optimization for loongson-3"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/loongson64/loongson3-memcpy.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/loongson64/loongson3-memcpy.S b/arch/mips/loongson64/loongson3-memcpy.S
index 4c0b6b8474f0..2021e75d5dd2 100644
--- a/arch/mips/loongson64/loongson3-memcpy.S
+++ b/arch/mips/loongson64/loongson3-memcpy.S
@@ -29,9 +29,9 @@
 
 #else /* LINUX_KERNEL */
 
+#include <linux/export.h>
 #include <asm/asm.h>
 #include <asm/asm-offsets.h>
-#include <asm/export.h>
 #include <asm/regdef.h>
 
 #define EXC(inst_reg,addr,handler)		\
-- 
2.48.1

